vite.config.ts

```diff
@@ -5,6 +5,13 @@ import devServer, { defaultOptions } from "@hono/vite-dev-server";
 import esbuild from "esbuild";
 import { flatRoutes } from "remix-flat-routes";
 import { cjsInterop } from "vite-plugin-cjs-interop";
+import { init } from "@paralleldrive/cuid2";
+
+const createHash = init({
+  length: 8,
+});
+
+const buildHash = process.env.BUILD_HASH || createHash();
```

Comment:
How do you plan to inject BUILD_HASH env?
Do you think it could be generated during some docker build?

Reply:
Thats the thing, I haven't thought of it yet, but I guess your suggestion is the correct way. I will add this and ask for a review again.